CMakeLists.txt

```diff
@@ -35,6 +35,13 @@ file(GLOB AWS_COMMON_SRC
         "source/*.c"
         )
 
+option(PERFORM_HEADER_CHECK "Performs compile-time checks that each header can be included independently. Requires a C++ compiler.")
+
+if (PERFORM_HEADER_CHECK) 
+    set(HEADERS_TO_CHECK ${AWS_COMMON_HEADERS} ${AWS_TEST_HEADERS})
+    add_subdirectory(cmake/header-tester)
```

Comment:
should the files from cmake/header-tester be in this review? Or are they being reviewed separately?

Reply:
I forgot to add these initially; they should be there now.